### PR TITLE
Enhancement/polls

### PR DESCRIPTION
**Pull request by k-awe-some**
- [x] Images are handled in poll cards and dialog.

### CODE REVIEW

File: client/src/pages/presentational/PollCard.jsx [hunk 1]
```diff
@@ -42,9 +47,7 @@ const PollCard = ({ question, images, pollId, isUser = false, deletePoll }) => {
         <Button>
           <DeleteForever onClick={() => deletePoll(pollId)} />
         </Button>
-      ) : (
-        <></>
-      )}
```
Comment (m3tron) [inline client/src/pages/presentational/PollCard.jsx:47]: this button should only appear if it's the user

Comment (k-awe-some) [reply client/src/pages/presentational/PollCard.jsx:47]: hmm ya all I did was change it to `null` so it'd look neat. any diff no?
this is the current snippet:
```
      {isUser ? (
        <Button>
          <DeleteForever onClick={() => deletePoll(pollId)} />
        </Button>
      ) : null}
```

Comment (m3tron) [reply client/src/pages/presentational/PollCard.jsx:47]: just saw the fix

**Review by m3tron — APPROVED** [review]

**Comment by k-awe-some** [comment]
will reopen when work on the edit feature for images

**Pull request merged**